src/translator/parser.h

```diff
@@ -23,9 +23,13 @@ inline marian::ConfigParser createConfigParser() {
       "--max-length-break", "Bergamot Options",
       "Maximum input tokens to be processed in a single sentence.", 128);
 
+  cp.addOption<bool>(
+      "--bytearray", "Bergamot Options",
+      "Enable loads through byte-arrays to enable testing of byte-array codepath (false by default)", false);
+
   cp.addOption<bool>(
       "--check-bytearray", "Bergamot Options",
-      "Flag holds whether to check the content of the bytearray (true by default)", true);
+      "Flag holds whether to check the content of the bytearray (false by default)", false);
```

Comment:
@qianqianzhu I changed the default behaviour here. Setting `--bytearray` to false and `--check-bytearray` default to true leads to validation issues. So to ensure that the cli always also validates the blobs, I manually change the parsed option to true in the other place (https://github.com/browsermt/bergamot-translator/pull/127/files#r626677517).

Reply:
Can we get a better solution. I don't like the hardcoded  `  options->set<bool>("check-bytearray", true);` even if it's just in the test binary.

Reply:
I see your point. What I meant https://github.com/browsermt/bergamot-translator/pull/127#discussion_r626677517 here is that we do not need to manually set `--check-bytearray` to true.

Reply:
@XapaJIaMnu Take a look now?